src/output.cpp

```diff
@@ -1146,6 +1146,23 @@ void popup_nowait( const char *mes, ... )
     popup( text, PF_NO_WAIT );
 }
 
+void popup_status( const char *title, const char *msg, ... )
```

Comment:
This functions seems to do pretty much the same as the main `popup` function.

It tests the first parameter being null, but it's always called with a non-null pointer anyway. The other pointer parameter is not tested, it is *expected* to be valid, why isn't the title expected to be valid? Why does the function require separate text and title strings?

It currently writes to `std::cerr` twice: here and in the main popup function. And it writes to `std::cerr` unconditionally, won't that mess up the output coming from ncurses?

Finally: `popup( text += fmt, ` - why `+=`, why not just `+`?